src/caret_analyze/infra/lttng/lttng.py

```diff
@@ -368,23 +467,77 @@ def _parse_lttng_data(
             begin = events[0][LttngEventFilter.TIMESTAMP]
             end = events[-1][LttngEventFilter.TIMESTAMP]
 
+            init_events = []
+            run_events = []
+
+            # 全てのトレースイベントを init_events, run_eventsに振り分ける
+            init_event_names = set(Lttng._prioritized_init_events)
             for event in events:
+                event_name = event[LttngEventFilter.NAME]
+                if event_name in init_event_names:
+                    init_events.append(event)
+                else:   # MYK test
+                    run_events.append(event)
+
+            # init_events をソートする
+            import functools
+            init_events.sort(key=functools.cmp_to_key(Lttng._compare_init_event))
+            # 初期化系トレースイベントを処理する
+            handler.create_init_handler_map()
+            for event in init_events:
+                event_name = event[LttngEventFilter.NAME]
+                if event_name in handler.handler_map:
+                    handler_ = handler.handler_map[event_name]
+                    handler_(event)
+
+            # ランタイム系トレースイベントを処理する
+            handler.create_runtime_handler_map()
+
+            for event in run_events:
                 if len(event_filters) > 0 and \
                         any(not f.accept(event, common) for f in event_filters):
                     continue
                 if store_events:
                     events.append(event)
                 filtered_event_count += 1
                 event_name = event[LttngEventFilter.NAME]
-                handler_ = handler.handler_map[event_name]
-                handler_(event)
+                if event_name in handler.handler_map:
+                    handler_ = handler.handler_map[event_name]
+                    handler_(event)
             data.finalize()
             if len(event_filters) > 0:
                 print('filtered to {} events.'.format(filtered_event_count))
 
         events_ = None if len(events) == 0 else events
         return data, events_, begin, end
 
+    @staticmethod
+    def _compare_init_event(
+        event1: dict,
+        event2: dict,
+    ) -> int:
+        """
+        Comparison function in sort function.
+
+        Returns
+        -------
+        str
+            1 if less, 0 if equal, and 1 if greater
+
+        """
```

Comment:
I suggest to remove this comment to prevent this method appeared in caret API list.
```suggestion
```

Reply:
Comment deleted